Update TN0037-api-gateways with Kong paper (#733)
apollographql/docs: src/content/technotes/TN0037-api-gateways.mdx

The key distinction of the router is that is not based on URLs or REST endpoints. The router is a GraphQL-native solution to handle your clients API operations.

> > Update: Apollo GraphQL and Kong have partnered to provide a joint paper to API practitioners with the tools they need to navigate the complex API landscape and drive successful digital experiences for their users. You can read more from the official [blog post](https://www.apollographql.com/blog/leveraging-graphql-for-next-generation-api-platforms)

Reviewer: ```suggestion
<Tip>

Apollo GraphQL and Kong have partnered to produce a joint paper that provides API practitioners with the tools they need to navigate the complex API landscape and drive successful digital experiences. Read more in the [blog post](https://www.apollographql.com/blog/leveraging-graphql-for-next-generation-api-platforms).

</Tip>
```
